src/main/java/com/lob/LobApiException.java

```diff
@@ -22,6 +25,22 @@ public ErrorResponse getErrorResponse() {
         return errorResponse;
     }
 
+    public void setStatusCode(int statusCode) {
+        this.statusCode = statusCode;
+    }
+
+    public int getStatusCode() {
+        return statusCode;
+    }
+
+    public FluentCaseInsensitiveStringsMap getHeaders() {
+        return headers;
```

Comment:
They are not immutable but a new one is created for each `Response` that comes from the API. The user will only be ever reading these values so I don't think a clone is necessary.